nautobot/extras/constants.py

```diff
@@ -15,3 +15,8 @@
     "statuses",
     "webhooks",
 ]
+
+# JobLogEntry Truncation Length
+MAX_GROUPING_LENGTH = 100
+MAX_LOG_OBJECT_LENGTH = 200
+MAX_ABSOLUTE_URL_LENGTH = 255
```

Comment:
Can we please prefix these setting names with `JOB_LOG_` so it's clear what they are for? Out of context it's not clear.
```suggestion
JOB_LOG_MAX_GROUPING_LENGTH = 100
JOB_LOG_MAX_LOG_OBJECT_LENGTH = 200
JOB_LOG_MAX_ABSOLUTE_URL_LENGTH = 255
```